<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Nastaveni</class>
 <widget class="QDialog" name="Nastaveni">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>367</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nastavení</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>320</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="Tloustka">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Počáteční tloušťka červů: </string>
   </property>
  </widget>
  <widget class="QLabel" name="Rychlost">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Rychlost pohybu:</string>
   </property>
  </widget>
  <widget class="QLabel" name="Barva">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Barva červů:</string>
   </property>
  </widget>
  <widget class="QLabel" name="CetMez">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Četnost mezer: </string>
   </property>
  </widget>
  <widget class="QLabel" name="CetBon">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>290</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Četnost bonusů (s):</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="TlSp">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>50</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="RySp">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>80</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="MezCb">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Mezery</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="BnSp">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>290</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="BonCb">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Bonusy</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="MezSp">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>210</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Body">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Vítězný počet bodů:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="BdSp">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="DobaTrvaniBon">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>290</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Doba trvání bonusů (s): </string>
   </property>
  </widget>
  <widget class="QSpinBox" name="DoBonSp">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>290</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="c1BarvaPb">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>110</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="c2BarvaPb">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>110</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Nastaveni</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Nastaveni</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
